docs/Authentication.md

```diff
@@ -178,4 +178,28 @@ to return an AAD token upon invoking the call method.
 The provider will be called for every request to the kusto service
 
 * **KUSTO_TOKEN_PROVIDER_CALLBACK_CLASSPATH**: 
-    'tokenProviderCallbackClasspath' - The classpath to the class
\ No newline at end of file
+    'tokenProviderCallbackClasspath' - The classpath to the class
+## Managed Identity Authentication
+This authentication method can use [managed identities](https://learn.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) to obtain Azure AD tokens without having to manage any credentials.
```

Comment:
Please explain here how to grant the managed identity access to Kusto

Reply:
Also refer to this
https://learn.microsoft.com/azure/data-explorer/configure-managed-identities-cluster?tabs=portal